src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -730,6 +730,7 @@ private function deleteFunction(Database $dbForConsole, callable $getProjectDB,
          */
         Console::info("Deleting VCS repositories and comments linked to function " . $functionId);
         $this->deleteByGroup('repositories', [
+            Query::equal('projectInternalId', [$projectInternalId]),
```

Comment:
```suggestion
            Query::equal('projectInternalId', [$project->getInternalId()]),
```

Variable is not defined in this scope. We can do exactly what we do here:  https://github.com/appwrite/appwrite/blob/fix-repository-deletion-bug/src/Appwrite/Platform/Workers/Deletes.php#L680